### PR TITLE
Feat/side drawer

### LINKED ISSUE #347: Create Log Details modal (UI only)
Show download icon on the right of "Logs" header.

Screenshot 2024-09-30 at 10 13 21

### PR DESCRIPTION
**Pull request by WilsonNet**
# Description

PR adding the sheet for the log excerpt 

Also adds a feature flag for features that are only visible in Dev Mode

https://www.figma.com/design/OY1Hp26cB4z3WyZPXxF8SE/KernelCI-Dashboard-V2?node-id=549-5434&node-type=frame&t=Voq6QoJzcPqpuFiS-0

## How to test

Go to any tree details in `Maestro`
Click on "Show Log" besides "Show More"

Open Sheet

image

CLoses #347

### CODE REVIEW

File: dashboard/src/components/Accordion/BuildAccordionContent.tsx [hunk 1]
```diff
@@ -207,13 +211,18 @@ const AccordionBuildContent = ({
...
+          <div className="flex flex-row gap-4">
+            <Button
+              variant="outline"
+              className="w-min rounded-full border-2 border-black text-sm text-dimGray hover:bg-mediumGray"
+              onClick={navigateToBuildDetails}
+            >
+              <FormattedMessage id="buildAccordion.showMore" />
+            </Button>
+            <DevOnly>
+              <SheetTrigger>Show Logs</SheetTrigger>
```
Comment (mari1912) [inline dashboard/src/components/Accordion/BuildAccordionContent.tsx:223]: nit: use locales

File: dashboard/src/locales/messages/index.ts [hunk 2]
```diff
@@ -139,6 +140,9 @@ export const messages = {
     'global.valid': 'Valid',
     'issue.failedToFetch': 'Failed to fetch issues',
     'issue.noIssueFound': 'No issue found.',
+    'logSheet.errorsFound': 'Errors found on Kernel.log',
+    'logSheet.fileName': 'File Name',
+    'logSheet.fileSize': 'File Name',
```
Comment (mari1912) [inline dashboard/src/locales/messages/index.ts:145]: file size?

File: dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  return (
+    <SheetContent className="w-[25rem] sm:w-full sm:max-w-[44rem]">
+      <SheetHeader className="mb-3">
+        <SheetTitle className="text-[1.75rem]">Build Logs</SheetTitle>
+      </SheetHeader>
+      <BaseCard className="gap-0" title={<>Logs</>}>
+        <div className="px-2 py-3 font-mono text-sm text-[#454545]">
+          Index of
+          /stable-rc/linux-5.10.y/v5.10.215-32-gd3c603576d90b/mips/32r2el_defconfig/gcc-10/logs/
+        </div>
```
Comment (mari1912) [inline dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx:36]: nit: todo, replace with real data

File: dashboard/src/components/Accordion/BuildAccordionContent.tsx (hunk 1 above)
Comment (WilsonNet) [reply dashboard/src/components/Accordion/BuildAccordionContent.tsx:224]: added todo

**Review by mari1912 — APPROVED** [review]

File: dashboard/src/hooks/useFeatureFlag.ts
[the remark this answers is not in the document]
Comment (WilsonNet) [reply dashboard/src/hooks/useFeatureFlag.ts:1]: Nice catch, fixed that

**Review by MarceloRobert — APPROVED** [review]

**Pull request merged**